crates/bevy_ecs/src/bundle.rs

```diff
@@ -62,11 +62,17 @@ use std::{any::TypeId, collections::HashMap};
 ///     y: Y,
 /// }
 ///
+/// // You have to implement `Default` for ignored field types in bundle structs.
+/// #[derive(Default)]
+/// struct Other(f32);
+///
 /// #[derive(Bundle)]
 /// struct B {
-///     #[bundle]
+///     #[bundle(nest)]
 ///     a: A,
 ///     z: Z,
+///     #[bundle(ignore)]
+///     other: Other,
```

Comment:
IMO we should use a `PhantomData` type + a bundle generic here, as it much more clearly motivates this feature.

Reply:
In the new PR I added the `PhantomData` but left this because I figured it demonstrates the `Default` trait requirement. Let me know if you want me to remove it.